lib/templates/structure/comment.php

```diff
@@ -38,4 +48,4 @@
 
 	beans_close_markup_e( 'beans_comment_body', 'div' );
 
-beans_close_markup_e( 'beans_comment', 'article' );
+	beans_close_markup_e( 'beans_comment', 'article' );
```

Comment:
Actually, this should be fully aligned left. Why?  It’s the closing `</article>` to line 10.

Reply:
According to WPCS, PHP indentation may not have any relation to nested HTML that it may be contained within.

Reply:
True. But we prefer to have this indentation in our template and fragment files.  Why? 

These files are directly related to the HTML structure that is sent out to the browser.  We run them through these functions for maximum flexibility.  Our intent with the indentation is to make it more readable.

At a glance, you can see the HTML structure.  Quickly you can see the opening and closing tags as we as what is within each.

And if you forget say a closing tag, this indentation pattern helps you to see it.  

If everything were left aligned, it would very easy to forget a closing tag.  It would also be a bit harder to see the final HTML structure.

Therefore, we prefer using the indentation to provide that visual representation of the HTML structure.